google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/CloudSdkAppEngineHelper.java

```diff
@@ -137,7 +137,7 @@ public AppEngineDeployAction createAutoDeploymentAction(
         artifactToDeploy,
         defaultAppYaml(),
         defaultDockerfile(artifactType),
-        AppEngineUtil.generateVersion(),
+        version,
```

Comment:
ah interesting.  I'm not crazy about that duplication.  How about we just make the versionId param to AppEngineDeployAction @NotNull?